keyserver/listener.go

```diff
@@ -100,6 +100,37 @@ func (server *ConiksServer) handleClientMessage(msg []byte) ([]byte, error) {
 
 	// handle request
 	switch req.Type {
+	default:
+		log.Printf("unknown message type: %q", req.Type)
+		response = p.NewErrorResponse(p.ErrorMalformedClientMessage)
+		err = p.Error(p.ErrorMalformedClientMessage)
+	}
+
+marshalling:
+	res, e := json.Marshal(response)
+	if e != nil {
+		panic(p.Error(p.ErrorInternalServer))
+	}
+	return res, err
+}
+
+func (server *ConiksServer) handleBotMessage(msg []byte) ([]byte, error) {
```

Comment:
Exactly, I was aware this issue, too. But I didn't find out any effective way to refactor this scope yet. I'm thinking about it.